docs/examples/201/policy-definition-with-assignment/main.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You'll need to run `bicep build` on the .bicep example you are checking in. We compare your generated JSON template to the one we generate in our tests to ensure you have transpiled your template with the latest version.